store/root.js

```diff
@@ -14,47 +14,45 @@ import apiRequest from 'api'
 import * as NetworkErrors from 'api/errors'
 
 class Root extends Store {
-  options = {
+  static defaultOptions = {
     allowAnonymousAccess: false,
-  }
 
-  constructor() {
-    const request = async (url, options) => {
+    request(url, options) {
```

Comment:
@viktor-yakubiv Can you create from this commit PR. It's a bug. I solved this issue slightly differently and I like your solution more. Also, we will avoid conflicts.